server/controllers/itemsController.js

```diff
@@ -27,14 +27,18 @@ module.exports = {
 	put: function(req, res) {
 		var itemID = req.params.itemID;
 		var newAttrs = req.body;
+    console.log(newAttrs);
```

Comment:
branch messed up, can we just merge it and then manually remove? or i'll pull down the master and make another PR